### DISCUSSION TITLE
Welcome to EEG-DL Discussions!

### DISCUSSION THREAD

**Discussion by SuperBruceJia** [SuperBruceJia/EEG-DL#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xiaoaixiaoai** [comment]
Hello, may I ask that the duration of the motor imagery data task phase read out is 125 seconds, but I don't really understand how to correspond to the task state, I hope to get your answers and help, thank you

**Reply by SuperBruceJia** [reply]
Hi, thanks a lot for your interest in our work.

Sorry for the late response. I encourage you to view the details from the website ([EEG Motor Movement/Imagery Dataset](https://archive.physionet.org/pn4/eegmmidb/)) or the original paper. Meanwhile, please check [this file](https://github.com/SuperBruceJia/EEG-DL/blob/master/Download_Raw_EEG_Data/Extract-Raw-Data-Into-Matlab-Files.py) line by line to understand the meaning.

If you have any questions, please let me know.

Best,

Shuyue
May 19, 2022

**Reply by ZhangGf94** [reply]
Many Thanks

**Reply by xiaoaixiaoai** [reply]
Okay.Sincerely thank you for your reply and suggestion

**Comment by dawin2015** [comment]
Thank you very much for your wonderful work. I recently read the EEG-related papers and found that the DL-related model has achieved more than 90% accuracy on a single data set, and the remaining 10% is something that I feel has landed in the industry. Therefore, at present, is it still meaningful to use similar graph neural networks to innovate models?

**Comment by ClareXUse** [comment]
may i ask a question that why there are two TransformerBlocks in the strcture of transformer?
the code is following
what is the difference between one layer and two layers?
# Encoder Architecture
transformer_block_1 = TransformerBlock(embed_dim=embed_dim, num_heads=num_heads, ff_dim=ff_dim)
transformer_block_2 = TransformerBlock(embed_dim=embed_dim, num_heads=num_heads, ff_dim=ff_dim)
x = transformer_block_1(x)
x = transformer_block_2(x)

**Comment by sangwan130** [comment]
i am using Read_Raw_Data_Save_Into_Matlab_Files.py for physionet dataset and error occurs is

X_105_C5, y_105_C5 = load_raw_data(electrodes=electrodes, subject=subject, num_classes=nclasses)
File " \Read_Raw_Data_Save_Into_Matlab_Files.py", line 200, in load_raw_data
return np.array(trials, dtype=np.float64).reshape((len(trials),) + trials[0].shape + (1,)), np.array(labels, dtype=np.float64)
IndexError: list index out of range

Can you help me to sort out this or provide link for dataset with labels
